charts/backbeat/templates/api/service.yaml

```diff
@@ -1,6 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+{{- if .Values.metrics.enabled }}
```

Comment:
In most charts, there's an `annotations` section in the `service` section of `values` where one can put arbitrary annotations. In our case, I'd fill this with the Prometheus SD annotations by default (unlike in most charts where this is empty), but follow the (naming) conventions of other charts.

Reply:
Do we want to have service level annotations or pod level annotations? What do you think it's best?

Reply:
Service-level. Don't do pod-level when there's a service whose endpoints can be resolved.